package.json

```diff
@@ -37,6 +37,7 @@
   },
   "private": true,
   "scripts": {
+    "createPatchTar": "tar -cvzf patch.tgz $(git diff --diff-filter=ACMRT -r --name-only --no-commit-id v1.8.2 | xargs)",
```

Comment:
a patch file would have been preferred IMO, anyway this script should not be commited